src/magic.c

```diff
@@ -1127,8 +1126,13 @@ double magic_resistance(unit * target)
     curse *c;
     const curse_type * ct_goodresist = 0, *ct_badresist = 0;
     const resource_type *rtype;
-    double probability = u_race(target)->magres;
+    const race *rc = u_race(target);
+    double probability = rc->magres;
+    const plane *pl = rplane(target->region);
 
+    if (rc == get_race(RC_HIRNTOETER) && !pl) {
+        probability /= 2;
+    }
     assert(target->number > 0);
     /* Magier haben einen Resistenzbonus vom Magietalent * 5% */
     probability += effskill(target, SK_MAGIC, 0) * 0.05;
```

Comment:
War mir nicht bewusst, dass es das bei allen Hirntötern ist. In spawn_braineaters bekommen die zumindest keine Magie zugewiesen (nur Tarnung und Wahrnehmung).